### ISSUE TITLE
Enable touch using Hammer.js. Fire drag events on press

### ISSUE THREAD

**Issue by shilpaprabhun** [BTMorton/angular2-grid#163]
I am using angular2-grid in my angular2 web application and it is working prefectly. Now I need to enable touch so that drag and drop can be done using mobile devices as well. To this end I have integrated Hammer.js.

Now, I need to fire onDragStart and onDragStop events on the Hammer.js' press and pressup events. How do I achieve this?

Here is a [sample ](http://stackoverflow.com/questions/36639486/angular2-manually-firing-click-event-on-particular-element)of firing one event inside of another. What is the equivalent on MouseEvent in angular-grid?

**Comment by BTMorton** [comment]
I've fixed the grid so that it will work with touch events now so hopefully you can use the regular mouse event stuff. I'll do a publish when I'm done with my current batch of changes and you can give it a go :)

**Comment by shilpaprabhun** [comment]
Ben, thanks very much for looking into this issue immediately.
I reviewed your commit and see the changes you have done with mouse and touch events. I am able to handle those events as well. I see you have updated your demo site (https://bmorton.co.uk/angular/) and that is functioning very nicely for mobile devices.
But, neither the mouse not the touch events are working in my application. As in, while I see they are being fired and I am able to handle them, the drag functionality is still not working. Can you please let me know what I am missing?

**Comment by BTMorton** [comment]
Yeah, there's a few other things I want to look into first, but I'll try and get it published by end of day today

**Comment by BTMorton** [comment]
This should be published. Is it working correctly for you now?
